raiden/network/proxies/token_network.py

```diff
@@ -338,7 +349,7 @@ def channel_exists_and_not_settled(
         )
         return exists_and_not_settled
 
-    def detail_participant(
+    def _detail_participant(
```

Comment:
why should this be private? this is exposing a public interface of the smart contract, not an internal implementation detail.

Reply:
The public interface of the smart contract is exposed by `detail_participants()`. The singular version is never used outside of this module.